packages/containers/psammead-timestamp-container/package.json

```diff
@@ -27,7 +27,9 @@
   "dependencies": {
     "@bbc/gel-foundations": "^3.4.0",
     "@bbc/psammead-timestamp": "^2.2.12",
-    "moment-timezone": "^0.5.26"
+    "@bbc/psammead-locales": "^2.23.0",
```

Comment:
Does this need to be in `dependencies` if it's only being used in the story? Shouldn't it just be a `devDependency`?

Because when storybook gets pushed to the live site the imports in the `index.stories.jsx` should resolve because `psammead-locales` is in the top level `package-lock.json` so `npm run ci:packages` should install it.

Reply:
I installed initially as a dev dependency, but if you would like to use this package in any project/application you would still need it as a dependency of this package to make it work. 

The user is not aware of what dependencies should be installed to make packages work. We add it to our top level `package.json` and `package-lock.json`, because we created it and we are aware of it.

Reply:
I installed it initially as a dev dependency, but if you would like to use this package in any project/application you would still need it as a dependency of this package to make it work. 

We add it to our top level `package.json` and `package-lock.json`, because we created it and know about it, but user should not be aware of what dependencies should be installed to make packages work.

Reply:
Would you need it as a normal dependency? The package should work fine because storybook is only really meant to be used for development and not a finished project/application. 

The only reason we have storybook available to view publically is so people can see the functionality of our components from my understanding.

Reply:
Let's say I want to use the `Timestamp` container in my application and I want to use the `hausa` locale. Just importing the `@bbc/psammead-local/ha` would work, without having to install `psammead-locales` in my project base level `package.json`.

Reply:
Discussed IRL and agreed to add it as a `devDependency`.